.github/workflows/test-e2e-single-chain.yml

```diff
@@ -1,77 +1,64 @@
-      uses: actions/checkout@v4
-      
-    - name: Checkout kurtosis-cdk
-      uses: actions/checkout@v4
-      with:
-        repository: 0xPolygon/kurtosis-cdk
-        path: kurtosis-cdk
-        ref: arpit/bridge-service
-    
-    - name: Install Kurtosis CDK tools
-      uses: ./kurtosis-cdk/.github/actions/setup-kurtosis-cdk
-    
-    - name: Setup Bats and bats libs
-      uses: bats-core/bats-action@2.0.0
-    
-    - name: Download aggkit archive
-      uses: actions/download-artifact@v4
-      with:
-        name: aggkit
-        path: /tmp
-    
-    - name: Load aggkit image
-      run: |
-        docker load --input /tmp/aggkit.tar
-        docker image ls -a
-
-    - name: Run E2E tests
-      run: make test-e2e-${{ matrix.e2e-group }}
-      working-directory: test
-      env:
-        KURTOSIS_FOLDER: ${{ github.workspace }}/kurtosis-cdk
-        BATS_LIB_PATH: /usr/lib/
-        AGGLAYER_PROVER_SP1_KEY: ${{ secrets.SP1_PRIVATE_KEY }}
-    
-    - name: Dump enclave logs
-      if: failure()
-      run: kurtosis dump ./dump
 
-    - name: Generate archive name
-      if: failure()
-      run: |
-        archive_name="dump_run_with_args_${{matrix.e2e-group}}_${{ github.run_id }}"
-        echo "ARCHIVE_NAME=${archive_name}" >> "$GITHUB_ENV"
-        echo "Generated archive name: ${archive_name}"
+  test-single-e2e:
+    name: Single chain E2E test
+    needs:
+      - build-aggkit-image
+    uses: agglayer/e2e/.github/workflows/aggkit-e2e-single-chain.yml@e0a65e7f56180c87f51ca50c0509f6d7c6cf1495
+    secrets: inherit
+    with:
+      docker-tag: local
+      kurtosis-cdk-ref: arpit/bridge-service
+      docker-artifact-name: aggkit
+      agglayer-e2e-ref: e0a65e7f56180c87f51ca50c0509f6d7c6cf1495
+      kurtosis-cdk-enclave-name: aggkit
+      kurtosis-cdk-args: |
+        {
+          "deployment_stages": {
+            "deploy_l2_contracts": true
+          },
+          "args": {
+            "verbosity": "debug",
+            "cdk_node_image": "aggkit:local",
+            "agglayer_image": "ghcr.io/agglayer/agglayer@sha256:5715855f2cc6834bd84a99a33937915164648547d21bfb55198948b0ae4e0fad",
+            "cdk_erigon_node_image": "hermeznetwork/cdk-erigon:v2.61.16-hotfix2",
+            "zkevm_contracts_image": "leovct/zkevm-contracts:v9.0.0-rc.5-pp-fork.12",
+            "zkevm_prover_image": "hermeznetwork/zkevm-prover:v8.0.0-RC16-fork.12",
+            "additional_services": [],
+            "consensus_contract_type": "pessimistic",
+            "sequencer_type": "erigon",
+            "erigon_strict_mode": false,
+            "gas_token_enabled": true,
+            "gas_token_address": "",
+            "agglayer_prover_sp1_key": "",
+            "zkevm_use_real_verifier": false,
+            "agglayer_prover_primary_prover": "mock-prover",
+            "enable_normalcy": true,
+            "zkevm_rollup_id": 1,
+            "binary_name": "aggkit",
+            "components": "aggsender,bridge"
+          }
+        }
```

Comment:
IMO it is cleaner if we could keep the Kurtosis CDK params in the yml file.  WDYT?

Reply:
will change